src/mesh/difops.cxx

```diff
@@ -452,6 +452,7 @@ const Field3D Div_par_LtoC(const Field3D &var) {
   Field3D result;
   result.allocate();
 
+  Mesh* mesh = var.getMesh();
```

Comment:
Should we get this `mesh` out before we create `result` so that we can do `Field3D result(mesh);`?